daffodil-core/src/main/scala/org/apache/daffodil/dsom/ElementBase.scala

```diff
@@ -352,6 +353,38 @@ trait ElementBase
     } else DataValue.NoValue
   }
 
+  /**
+   * Is either None or Some(primTypeValue).
```

Comment:
This comment isn't accurent. This is always a pimitive, but might be the special NoValue.

Reply:
I had noticed that the comment seemed inaccurate when I copied it too.  Yes, I'll update both places.